appveyor.yml

```diff
@@ -7,7 +7,7 @@ environment:
   access_token:
     secure: ZxcrtxQXwszRYNN6c1ZIagczEqzmQQZeYHY58izcmF0jdq/cptxJvFUoVxDmnoqj
 install:
-  - ps: New-SelfSignedCertificate -DnsName localhost -CertStoreLocation cert:Localmachine\My
+  - ps: Import-Certificate -FilePath test/schannel-ca.crt -CertStoreLocation 'Cert:\CurrentUser\Root' -Verbose
```

Comment:
This will be very annoying...
UAC doesn't allow that, since no "arbitrary" program should be able to touch the root certificate store.
So with any solution I know, the confirmation dialog pops up.